openmc/mgxs/library.py

```diff
@@ -524,6 +524,15 @@ def add_to_tallies_file(self, tallies_file, merge=True):
         for domain in self.domains:
             for mgxs_type in self.mgxs_types:
                 mgxs = self.get_mgxs(domain, mgxs_type)
+
+                if mgxs_type in openmc.mgxs.MDGXS_TYPES:
+                    if self.num_delayed_groups == 0:
+                        mgxs.delayed_groups = None
+                    else:
+                        delayed_groups \
```

Comment:
Will do. I changed it to the following so it would fit on two lines and be less than 80 characters
    
    mgxs.delayed_groups \
        = list(range(1, self.num_delayed_groups+1))